src/psyclone/psyir/backend/sympy_writer.py

```diff
@@ -208,32 +351,51 @@ def convert_to_sympy_expressions(list_of_expressions):
         :type list_of_expressions: list of \
             :py:class:`psyclone.psyir.nodes.Node`
 
-        :returns: the converted PSyIR expressions.
-        :rtype: list of SymPy expressions
-
-        '''
+        :returns: a 2-tuple consisting of the the converted PSyIR \
```

Comment:
No need for line-continuation symbols ;-)

Reply:
Ah yes :)